src/pages/iou/request/step/IOURequestStepScan/NavigationAwareCamera/index.tsx

```diff
@@ -1,48 +1,29 @@
-import PropTypes from 'prop-types';
 import React, {useEffect, useRef} from 'react';
+import type {ForwardedRef} from 'react';
 import {View} from 'react-native';
 import Webcam from 'react-webcam';
 import useTabNavigatorFocus from '@hooks/useTabNavigatorFocus';
-
-const propTypes = {
-    /** Flag to turn on/off the torch/flashlight - if available */
-    torchOn: PropTypes.bool,
-
-    /** The index of the tab that contains this camera */
-    cameraTabIndex: PropTypes.number.isRequired,
-
-    /** Callback function when media stream becomes available - user granted camera permissions and camera starts to work */
-    onUserMedia: PropTypes.func,
-
-    /** Callback function passing torch/flashlight capability as bool param of the browser */
-    onTorchAvailability: PropTypes.func,
-};
-
-const defaultProps = {
-    onUserMedia: undefined,
-    onTorchAvailability: undefined,
-    torchOn: false,
-};
+import type {NavigationAwareCameraProps} from './types';
 
 // Wraps a camera that will only be active when the tab is focused or as soon as it starts to become focused.
-const NavigationAwareCamera = React.forwardRef(({torchOn, onTorchAvailability, cameraTabIndex, ...props}, ref) => {
-    const trackRef = useRef(null);
+function NavigationAwareCamera({torchOn, onTorchAvailability, cameraTabIndex, ...props}: NavigationAwareCameraProps, ref: ForwardedRef<Webcam>) {
```

Comment:
```suggestion
function NavigationAwareCamera({torchOn, onTorchAvailability, cameraTabIndex, ...props}: NavigationAwareCameraProps, ref: ForwardedRef<Webcam | Camera>) {
```

Reply:
And below I think you need to assert when passing ref like `ref as unknown as Webcam`